fixing cloud shell errors

jluk opened the pull request:
Fix product name, direct users to skip install/setup for cloud shell. Az login is disabled in cloud shell, "cliinteractive" should never be used on az login cmds as users are already authenticated in cloud shell.

azurecla commented:
Hi __@Juliako__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (Juliako). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

jluk committed:
fixing cloud shell errors

Fix product name, direct users to skip install/setup for cloud shell. Az login is disabled in cloud shell, "cliinteractive" should never be used on az login cmds as users are already authenticated in cloud shell.

echuvyrov reviewed (approved):
LGTM!

GitHubber17 committed:
Update terraform-install-configure.md

GitHubber17 committed:
Update terraform-install-configure.md
